### DISCUSSION TITLE
Fedora | A small fix in case libalac (or other libraries installed in /usr/local/ can't be found.

### DISCUSSION THREAD

**Discussion by thiccaxe** [mikebrady/shairport-sync#1762] [General]
Follow the instructions in the libalac repo (`ldconfig`)

Then, back in shairport, run autoreconf as normal. When you run `./configure flags=abc`, modify it as such with the pkc config environment variable:
```sh
PKG_CONFIG_LIBDIR=/usr/local/lib/pkgconfig:/usr/lib/pkgconfig/:/lib64/pkgconfig:/lib/pkgconfig ./configure --flags --with-apple-alac=/usr/local
```
Run make as usual.

To run shairport, you must set the linking library path environment variable as so:
`LD_LIBRARY_PATH=/usr/local/lib ./shairport-sync --flags`

note for pkg config, on other distributions, libraries might be installed elsewhere, simply add that to PKG_CONFIG_LIBDIR. To find out where the libraries are installed, you can run `ldconfig -p` as such:
```sh
->  ldconfig -p | grep libpopt
        libpopt.so.0 (libc6,x86-64) => /lib64/libpopt.so.0
        libpopt.so (libc6,x86-64) => /lib64/libpopt.so
```
note that this will likely not show your user-installed libraries such as libalac or libplist (which I recommend installing from source, as Fedora only has the 2.2.0 version, which doesn't work with shairport sync)

**Comment by mikebrady** [comment]
Many thanks for this.
